code/mod_table/mod_table.cpp

```diff
@@ -106,6 +107,10 @@ void parse_mod_table(const char *filename)
 			mprintf(("Game settings table: Use tabled strings (translations) for the default language: %s\n", Use_tabled_strings_for_default_language ? "yes" : "no"));
 		}
 
+		if (optional_string("$Process AI before physics:")) {
+			stuff_boolean(&Ai_before_physics);
+		}
```

Comment:
On a second look, I can't find where you set this bool's default value. `Use_engine_wash_intensity` is set in `mod_table_reset` in `mod_table.cpp`, for instance.